example-data/turtle/sio-model/consent.ttl

```diff
@@ -0,0 +1,22 @@
+@prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix edam: <http://edamontology.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+:identifier_ a edam:data_0842 ;
```

Comment:
I think we can remove `:identifier_` node from this turtle file. Since we don't use this node in all other example turtle files. Also it makes the figure a bit compact.

Reply:
I can do that. The identifier nodes are there because they were also in the previous TTL examples I updated ;) . But I agree that could be removed.